config/samples/custom-theme/README.md

```diff
@@ -0,0 +1,274 @@
+# Customizing Horizon Dashboard Theme with horizon-operator
+
+The horizon-operator provides mechanisms to customize the Horizon Dashboard
+theme. A theme is a directory containing assets that override the default
+styling of the dashboard. Horizon can be configured to support multiple themes
+simultaneously, making them available at runtime. Through the horizon-operator,
+you can load custom themes using the
+[ExtraMounts](https://github.com/openstack-k8s-operators/dev-docs/blob/main/extra_mounts.md)
+feature.
+
+## Theme configuration
+
+To provide additional themes, you must specify them in the `AVAILABLE_THEMES`
+setting within the `_11_rcue_theme.py` file. Themes are defined as a list of
+tuples with the following format:
+
+```
+('name', 'label', 'path')
+```
+
+Where:
+
+- name: The key by which the theme value is resolved
+- label: The display name or description of the custom theme
+- path: The directory location where the theme is unpacked and stored
+
+Currently, the `horizon-operator` only supports the default path:
+`/usr/share/openstack-dashboard/openstack_dashboard/themes`
+
+## Procedure
+
+To use a custom theme, create a `_11_rcue_theme.py` file and configure the
+`AVAILABLE_THEMES` as follows:
+
+```python
+# override the CUSTOM_THEME_PATH variable with this settings snippet
+AVAILABLE_THEMES = [('rcue', 'Custom Theme', 'themes/rcue')]
+```
+
+In case of multiple themes, additional entries in the AVAILABLE_THEMES tuple can
+be provided:
+
+```python
+# override the CUSTOM_THEME_PATH variable with this settings snippet
+AVAILABLE_THEMES = [('rcue', 'Custom Theme', 'themes/rcue'), ('rcue_alt', 'Custom Alt Theme', 'themes/rcue_alt')]
+```
+
+> **Note**: If `AVAILABLE_THEMES` contains only a single theme tuple, the theme
+toggle functionality will not be available in the dashboard UI.
+
+
+The configuration file must be mounted in
+`/etc/openstack-dashboard/local_settings.d` within the Pod. The horizon
+bootstrap process uses this file to resolve the theme path during the `python
+manage.py collectstatic` operation.
+
+After creating the `_11_rcue_theme.py` file, you must provide the theme assets
+as a **tar.gz** tarball. Horizon will unpack this tarball in
```

Comment:
Do you want to change `Horizon will unpack` to `A script in the Horizon operator will unpack`?

Reply:
sure, I'll change that in my next iteration

Reply:
Done